src/Microsoft.ML.PipelineInference/AutoInference.cs

```diff
@@ -536,7 +539,26 @@ public PipelinePattern[] GetNextCandidates(int numberOfCandidates)
                 var currentBatchSize = numberOfCandidates;
                 if (_terminator is IterationTerminator itr)
                     currentBatchSize = Math.Min(itr.RemainingIterations(_history), numberOfCandidates);
-                BatchCandidates = AutoMlEngine.GetNextCandidates(_sortedSampledElements.Select(kvp => kvp.Value), currentBatchSize);
+                BatchCandidates = AutoMlEngine.GetNextCandidates(
+                    _sortedSampledElements.Select(kvp => kvp.Value), 
+                    currentBatchSize,
+                    this._columnPurpose);
+
+                var h = _env.Register("AutoMlMlState");
+                using (var ch = h.Start("GetNextCandidates"))
+                {
+                    foreach (var pipeline in BatchCandidates)
+                    {
+                        ch.Info("AutoInference Suggested Transforms.");
```

Comment:
>AutoInference Suggested Transforms. [](start = 33, length = 35)

either add pipeline identification, or remove it, what's  the point to repeat same phrase all the time?